docs/build/EVM/precompiles/xcm/native-transfer.md

```diff
@@ -0,0 +1,74 @@
+# Transfer native token
+
+Let's use `transfer_multiasset` to:
+
+- transfer native token of `parachainId` **2000** to a Wrapped version in **2007**
+- for amount **10000000000000000000**
+
+#### 1. Define call as payable
+
+As the call will be on behalf of the contract, the native amount should be held by the contract. Please make the function payable to ensure the native token will be transferred to contract.
+
+```solidity
+    function transfer_native() external payable {
+```
+
+#### 2. Asset Multilocation
+
+The Native token Multilocation is defined by: `Multilocation: { parents: 0, interior: Here }`
+The interior field is an empty bytes array (equivalent of `Here`).
+
+```solidity
+bytes[] memory interior1 = new bytes[](0);
```

Comment:
have you tested all of these functions?

Reply:
YEs ofc. here: https://github.com/AstarNetwork/EVM-XCM-Examples/pull/2